### PR TITLE
chore: WearableController/FacialFeatureController test suite

### PR DESCRIPTION
**Pull request by AjimenezDCL**
#1361 A new tests suite independent from `AvatarRenderer` to increase the coverage on this feature

### CODE REVIEW

**Review by pravusjif — APPROVED** [review]
Also tested fiddling around with the avatar editor in the build, and loading/re-loading avatars, everything worked fine

File: unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        private WearableDictionary catalog;
+
+        //TODO: Remove it once we have a mocking library
+        private class Mock_BodyShapeController : IBodyShapeController
+        {
+            public string bodyShapeId => WearableLiterals.BodyShapes.FEMALE;
+            public void SetupEyes(Material material, Texture texture, Texture mask, Color color) { }
+            public void SetupEyebrows(Material material, Texture texture, Color color) { }
+            public void SetupMouth(Material material, Texture texture, Color color) { }
+        }
```
Comment (sandrade-dcl) [inline unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs]: Couldn't we use our new mocking library NSubstitute here?

**Review by sandrade-dcl — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs]: Yeah, this PR is older than the one where NSubstitue was integrated. Gonna put the branch up to date and change it, nice catch

**Review by sandrade-dcl — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs (hunk 1 above)
Comment (sandrade-dcl) [reply unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Tests/FacialFeatureControllerShould.cs]: Beautiful! Thanks! :)

**Pull request merged**